tests/unit/test_filesystem.py

```diff
@@ -10,68 +10,62 @@
-            self.assertTrue(fd)
-            os.close(fd)
-            self.assertTrue(lock_file(path, timeout=0))
-
 
 class FakeFilesystemTest(fake_filesystem_unittest.TestCase):
     def setUp(self):
         self.setUpPyfakefs()
 
     def test_clean_directory_creates_parents(self):
-        clean_directory('/tmp/dirty-dir')
+        clean_directory(os.path.join('tmp', 'dirty-dir'))
 
-        self.assertTrue(os.path.exists('/tmp/dirty-dir'))
-        self.assertFalse(os.listdir('/tmp/dirty-dir'))
+        self.assertTrue(os.path.exists(os.path.join('tmp', 'dirty-dir')))
+        self.assertFalse(os.listdir(os.path.join('tmp', 'dirty-dir')))
 
     def test_clean_directory_removes_contents(self):
-        self.fs.CreateFile('/tmp/dirty-dir/file1')
-        self.fs.CreateFile('/tmp/dirty-dir/file2')
-        self.fs.CreateDirectory('/tmp/dirty-dir/dir1')
-        self.fs.CreateFile('/tmp/dirty-dir/dir1/file3')
-        clean_directory('/tmp/dirty-dir')
+        self.fs.CreateFile(os.path.join('tmp', 'dirty-dir', 'file1'))
+        self.fs.CreateFile(os.path.join('tmp', 'dirty-dir', 'file2'))
+        self.fs.CreateDirectory(os.path.join('tmp', 'dirty-dir', 'dir1'))
+        self.fs.CreateFile(os.path.join('tmp', 'dirty-dir', 'dir1', 'file3'))
+        clean_directory(os.path.join('tmp', 'dirty-dir'))
 
-        self.assertTrue(os.path.exists('/tmp/dirty-dir'))
-        self.assertFalse(os.listdir('/tmp/dirty-dir'))
+        self.assertTrue(os.path.exists(os.path.join('tmp', 'dirty-dir')))
+        self.assertFalse(os.listdir(os.path.join('tmp', 'dirty-dir')))
 
     def test_clean_file_creates_parents(self):
-        clean_file('/tmp/dirty-dir/file')
+        clean_file(os.path.join('tmp', 'dirty-dir', 'file'))
 
-        self.assertTrue(os.path.exists('/tmp/dirty-dir'))
-        self.assertFalse(os.listdir('/tmp/dirty-dir'))
+        self.assertTrue(os.path.exists(os.path.join('tmp', 'dirty-dir')))
+        self.assertFalse(os.listdir(os.path.join('tmp', 'dirty-dir')))
 
     def test_clean_file_removes_file(self):
-        self.fs.CreateFile('/tmp/dirty-dir/file1')
-        self.fs.CreateFile('/tmp/dirty-dir/file2')
-        self.fs.CreateDirectory('/tmp/dirty-dir/dir1')
-        self.fs.CreateFile('/tmp/dirty-dir/dir1/file3')
-        clean_file('/tmp/dirty-dir/file1')
+        self.fs.CreateFile(os.path.join('tmp', 'dirty-dir', 'file1'))
+        self.fs.CreateFile(os.path.join('tmp', 'dirty-dir', 'file2'))
+        self.fs.CreateDirectory(os.path.join('tmp', 'dirty-dir', 'dir1'))
+        self.fs.CreateFile(os.path.join('tmp', 'dirty-dir', 'dir1', 'file3'))
+        clean_file(os.path.join('tmp', 'dirty-dir', 'file1'))
 
-        self.assertTrue(os.path.exists('/tmp/dirty-dir'))
-        self.assertEqual(set(os.listdir('/tmp/dirty-dir')), set(['file2', 'dir1']))
+        self.assertTrue(os.path.exists(os.path.join('tmp', 'dirty-dir')))
+        self.assertEqual(set(os.listdir(os.path.join('tmp', 'dirty-dir'))), set(['file2', 'dir1']))
```

Comment:
I think these might be fine on a real filesystem, but pyfakefs is strict. And I think that's probably a good thing.